src/services/exile_service.py

```diff
@@ -76,7 +74,9 @@ async def exile_user(
             logging_embed, logger, f"Failed to send DM to exiled user, {e}"
         )
 
-    log_info_and_embed(logging_embed, logger, f"<@{user.id}> was successfully exiled")
+    log_info_and_embed(
+        logging_embed, logger, f"<@{user.id}> was successfully exiled for {duration}"
```

Comment:
while we're here i think we can also include an end time with the unix timestamp like on line 70